subsys/net/ip/net_mgmt.c

```diff
@@ -34,7 +34,7 @@ struct mgmt_event_wait {
 };
```

Comment:
There is unnecessary file path set in commit subject `subsys/net/ip/net_mgmt: Use mutex for net_mgmt_lock`
Just say `net: mgmt: .....`

Reply:
Didn't really know what should I put, usually I just follow the path, thanks for the info

Reply:
FYI, the commit subject should identify the logical component that the commit is changing. It is pointless to put file path there as that can be seen from the commit itself.

Reply:
Also now you have `net: net_...` which looks funny. Please do `net: mgmt: ...` instead, we have only one mgmt API in net so no need to repeat the file / API prefix ("net_") there.

Reply:
Fixed, thanks!